### ISSUE TITLE
Error when updating to the latest Angular Version

### ISSUE THREAD

**Issue by enniob** [snics/ng2-avatar#13]
### Bug Report or Feature Request (mark with an `x`)
```
- [X] bug report -> please search issues before submitting
- [ ] feature request
```

### OS and Version?
Windows Server 2016

### Versions
Angular CLI: 6.2.3
Node: 8.9.4
OS: win32 x64
Angular: 6.1.8
... animations, common, compiler, compiler-cli, core, forms
... http, language-service, platform-browser
... platform-browser-dynamic, router

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.6.8
@angular-devkit/build-angular     0.6.8
@angular-devkit/build-optimizer   0.6.8
@angular-devkit/core              0.6.8
@angular-devkit/schematics        0.8.3
@angular/cdk                      6.4.7
@angular/cli                      6.2.3
@angular/material                 6.4.7
@ngtools/webpack                  6.0.8
@schematics/angular               0.8.3
@schematics/update                0.8.3
rxjs                              6.3.2
typescript                        2.9.2
webpack       4.8.3

npm 5.6.0

node v8.9.4

### Repro steps
ng update @angular/core

### The log given by the failure
 Package "ng2-avatar" has an incompatible peer dependency to "@angular/core" (requires "^2.0.0" (extended), would install "6.1.9").
                  Package "ng2-avatar" has an incompatible peer dependency to "@angular/common" (requires "^2.0.0" (extended), would install "6.1.9").
Incompatible peer dependencies found. See above.

**Comment by snics** [comment]
Hi @enniob, thanks to you for the report. I'm going to look at the error on sunday as I'm still on vacation. But I think this is just a small bug because I don't use any bigger APIs from the two angular modules.

**Comment by pedrobacchini** [comment]
Any solution?

**Comment by enniob** [comment]
@pedrobacchini I never heard back, so I end up making my own version of it.

**Comment by kfirufk** [comment]
@pedrobacchini - have you made your own version ?

**Comment by enniob** [comment]
@pedrobacchini yes, I made a simpler version. Look at my profile and you can see it. If you need any updates let me know and I can add to it
